pkg/protocol/http.go

```diff
@@ -32,12 +34,12 @@ func (h *HTTPHandler) HandleRequest(r io.ReadCloser, w io.WriteCloser, netReques
 	for {
 		req, err := http.ReadRequest(bufioHTTPReader)
 		if err == io.EOF {
-			log.Print("EOF while parsing request HTTP")
+			h.logger.Error("EOF while parsing request HTTP")
```

Comment:
actually this is normal, I think this is debug level. We get it each time connection is closed.